Gopkg.toml

```diff
@@ -0,0 +1,3 @@
+[prune]
+  go-tests = true
```

Comment:
If we're going with go dep (see other comment), I would prefer the config `non-go = true` to reduce the weight of the vendor folder.

Reply:
I'd done that at first, but the changes were more severe 😉.

I'll alter that for now.

Reply:
No worries about that :)